content/practices/practices.html

```diff
@@ -75,6 +75,18 @@ <h2 class="tile-name">
         </div>
       </li>
 
+      <li class="tile">
+        <a href="high-contrast/high-contrast-practice.html">
```

Comment:
```suggestion
        <a href="color-settings/color-settings-practice.html">
```

@mcking65 needed change because of the folder rename in https://github.com/w3c/aria-practices/pull/2991/commits/3d7cc355895bb34cb6b391143cf2f516017c60f1